CommonData/databaseinterface.cpp

```diff
@@ -1092,7 +1092,12 @@ bool DatabaseInterface::columnGetComputedInfo(int columnId, int &analysisId, boo
 		std::string constructorJsonStr	= _wrap_sqlite3_column_text(stmt,	5);
 					analysisId			= sqlite3_column_int(		stmt,	6);
 
-		codeType = codeTypeStr.empty() ? computedColumnType::notComputed : computedColumnTypeFromString(codeTypeStr);
+		codeType = computedColumnType::notComputed;
+		if (!codeTypeStr.empty())
+		{
+			try { codeType = computedColumnTypeFromString(codeTypeStr); }
+			catch(...) {}
+		}
```

Comment:
```suggestion
		try { codeType = computedColumnTypeFromString(codeTypeStr); }
		catch(...) {}
```

There is no need for the if, if it is empty it will also fail and thus get caught

Reply:
Yes but I saw that in the file that was given in the issue, that most of the columns have empty value. If there are hundreds of columns, that would make hundreds of catch exceptions, which can be a bit long.

Reply:
Ok well I dont think a few hundred exceptions will make a difference but whatever.